src/layers/ZweitausbildungPoisLayer/ZweitausbildungPoisLayer.js

```diff
@@ -0,0 +1,204 @@
+import VectorLayer from 'react-spatial/layers/VectorLayer';
+import OLVectorLayer from 'ol/layer/Vector';
+import OLVectorSource from 'ol/source/Vector';
+import ClusterSource from 'ol/source/Cluster';
+import GeoJSON from 'ol/format/GeoJSON';
+import Style from 'ol/style/Style';
+import Icon from 'ol/style/Icon';
+import Text from 'ol/style/Text';
+import Fill from 'ol/style/Fill';
+import Circle from 'ol/style/Circle';
+import Stroke from 'ol/style/Stroke';
+import { getVectorContext } from 'ol/render';
+import { unByKey } from 'ol/Observable';
+import { easeOut } from 'ol/easing';
+
+/**
+ * Layer for zweitausbildung pois
+ * Extends {@link https://react-spatial.geops.de/docjs.html#layer geops-spatial/Layer}
+ * @private
+ * @class
+ * @param {Object} [options] Layer options.
+ */
+class ZweitausbildungPoisLayer extends VectorLayer {
+  constructor(options = {}) {
+    const olLayer = new OLVectorLayer({
+      style: f => this.style(f),
+    });
+
+    super({
+      ...options,
+      olLayer,
+    });
+
+    this.styleCache = {};
+    this.setVisible(this.visible);
+
+    this.zweitProps = this.get('zweitausbildung') || {};
+    this.color = this.zweitProps.color || 'rgba(0, 61, 133, 0.8)';
+    const viewparams = this.zweitProps.viewparams
+      ? `viewparams=${this.zweitProps.viewparams}&`
+      : '';
+
+    this.source = new OLVectorSource({
+      format: new GeoJSON(),
+      loader: () => {
+        fetch(
+          `${this.geoServerUrl}?` +
+            'service=WFS&version=1.0.0&request=GetFeature&' +
+            'typeName=trafimage:zweitausbildung_pois_qry&' +
+            `srsName=EPSG:3857&${viewparams}` +
+            'outputFormat=application/json',
+        )
+          .then(data => data.json())
+          .then(data => {
+            const format = new GeoJSON();
+            const features = format.readFeatures(data);
+            this.source.clear();
+            this.source.addFeatures(features);
+          });
+      },
+    });
+
+    olLayer.setSource(
+      new ClusterSource({
+        distance: 70,
+        source: this.source,
+      }),
+    );
+
+    this.animate = this.animate.bind(this);
+  }
+
+  setGeoServerUrl(geoServerUrl) {
+    this.geoServerUrl = geoServerUrl;
+  }
+
+  init(map) {
+    super.init(map);
+
+    this.olLayerHighlight = new OLVectorLayer({
+      map,
+      source: new OLVectorSource(),
+    });
+
+    this.olLayerHighlight.getSource().on('addfeature', () => {
+      this.animationStartTime = new Date().getTime();
+      unByKey(this.postrenderKey);
+      this.postrenderKey = this.olLayerHighlight.on('postrender', this.animate);
```

Comment:
Unfortunately, there is the default style (small blue circle) below the grey animation circle. And I haven't found out how I can avoid this. When I set a style function for the olLayerHight or directly to the highlightFeature, the postrender callback isn't called. @oterral any ideas?

Reply:
Fixed